media/js/firefox/releasenotes.js

```diff
@@ -11,16 +11,44 @@
 
     var stickyNav = new Waypoint.Sticky({
         element: $nav,
-        handler: function(direction){
-            if(direction == 'down'){
+        handler: function(direction) {
+            if (direction === 'down'){
                 $nav.addClass('fixedNav');
             } else{
                 $nav.removeClass('fixedNav');
             }
-
         }
     });
 
+    // navigation drop-down menu
+    function initSubMenu() {
+        var $subMenu = $('.submenu-title');
+        var $subMenuLinks = $subMenu.find('.submenu > li > a');
+        var $subMenuTitle = $subMenu.find('> a');
+
+        $subMenu.on('mouseenter', function() {
+            $subMenu.attr('aria-expanded', true);
+        });
+
+        $subMenu.on('mouseout', function() {
+            $subMenu.attr('aria-expanded', false);
+        });
+
+        $subMenuTitle.on('focus', function() {
+            $subMenu.attr('aria-expanded', true);
+        });
+
+        $subMenuTitle.on('click', function(e) {
+            e.preventDefault();
+        });
+
+        $subMenuLinks.on('focusout', function(e) {
+            if ($(e.target).parent().is('li:last-child')) {
```

Comment:
Small issue here - if you tab backwards/shift-tab from the last `<li>`, the menu disappears. Maybe needs a `$subMenuLinks.on('focus')` handler?